test/sequential/xyk-pallet.api.maintenance-rules.test.ts

```diff
@@ -33,12 +33,16 @@ import { Maintenance } from "../../utils/Maintenance";
 import { getLiquidityAssetId } from "../../utils/tx";
 import { ProofOfStake } from "../../utils/ProofOfStake";
 import { User } from "../../utils/User";
+import { Market } from "../../utils/market";
 
 jest.spyOn(console, "log").mockImplementation(jest.fn());
 jest.setTimeout(1500000);
 
 let users: User[] = [];
 let tokenIds: BN[] = [];
+const poolIds: BN[] = [];
+let liqId: BN;
+let i: number = 0;
```

Comment:
why those new stuff?

Reply:
Because we use poolIds in new pallet instead of tokenIds and we need to receive them